blockchain/contracts/StakingNFT.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol";
+import "@openzeppelin/contracts/interfaces/IERC721.sol";
+import "@openzeppelin/contracts/interfaces/IERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract StakingContract is IERC721Receiver {
+    IERC721 public nft;
+
+    struct Stake {
+        uint256 tokenId;
+        uint256 amount;
```

Comment:
What is the purpose of `amount` here?

Reply:
was needed during my first implementation (when you could stake multiple NFTs), i'll update the PR with the new modifer & remove amount